Composer/packages/client/src/pages/about/index.tsx

```diff
@@ -76,32 +76,14 @@ export const About: React.FC<RouteComponentProps> = () => {
         </div>
         <div css={about.linkContainer}>
           <div css={about.linkRow}>
-            <IconButton
-              styles={about.icon}
-              ariaLabel={formatMessage('Terms of User Button')}
-              iconProps={{ iconName: 'BlockedSite' }}
-              tabIndex={-1}
-            />
-            <Link
-              href={'https://github.com/microsoft/BotFramework-Composer/blob/stable/LICENSE.md'}
-              tabIndex={-1}
-              target={'_blank'}
-            >
+            <Icon styles={about.icon} iconName={'BlockedSite'} tabIndex={-1} />
```

Comment:
Does `<Link>` allow an icon to be a part of the text? it looks wonky atm: 
![image](https://user-images.githubusercontent.com/14900841/79376458-10202300-7f0f-11ea-9c96-5ee5c43e30c3.png)

Reply:
Link doesn't, but I agree it looks wonky. I'll fix that up.